src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private String operation;
```

Comment:
It is better to do Enum
```suggestion
    private Operation operation;
```